docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -169,12 +169,15 @@ If you are using an *on-premises* Elastic Stack deployment with <<using-kibana-w
 
 To access alerting in a space, a user must have access to one of the following features:
 
+* Alerting
 * <<xpack-apm,*APM*>>
+* <<logs-app,*Logs*>>
 * <<metrics-app,*Metrics*>>
 * <<xpack-siem,*Security*>>
 * <<uptime-app,*Uptime*>>
 
-See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to these features. 
+See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to these features.
+Also note that a user will need privileges for the *Actions and Connectors* feature to attach actions to an alert.
```

Comment:
Might be worth adding that you'd need *Actions and Connectors*  to modify an Alert that already has actions attached to it.
(Also, should we clarify that `read` privilege is enough for that?)

Reply:
Yup makes sense to me, added better explanation.